### PR TITLE
Sensitive fix

### PR DESCRIPTION
**Pull request by rwwh**
As described in slack, a partial fix for mixups in this text. Some decisions of scope are needed.

### CODE REVIEW

**Comment by bedroesb** [comment]
@korbinib is this also a good time to have a look at this issue #576 ?

File: pages/your_problem/sensitive_data.md [hunk 1]
```diff
@@ -10,18 +10,22 @@ page_tag: sensitive
...
+This can for example be personal or commercial information, but also information such as breeding grounds of endangered species. 
+Any such data must be protected against unauthorized access.
+What is considered sensitive information is usually regulated by national laws and may differ between countries. You should be cautious when you are dealing with sensitive, or potentially sensitive, information.
+
 
 ### Considerations
 
-* If you deal with any information about individuals from the EU, you are bound by the GDPR and such data is called “personal data”.
-* Not all personal information is sensitive: The GDPR [Article 9] identifies specific pieces of data as sensitive: data consisting of racial or ethnic origin, political opinions, religious or philosophical beliefs, or trade union membership, genetic data, biometric data, data concerning health or data concerning a natural person's sex life or sexual orientation.  
+* If you deal with any information about individuals from the EU, you are bound by the General Data Protection Regulation (GDPR), such data is called “personal data”.
```
Comment (nsoranzo) [inline pages/your_problem/sensitive_data.md:21]:
```suggestion
* If you deal with any information about individuals from the EU, you are bound by the [General Data Protection Regulation (GDPR)](https://gdpr.eu/what-is-gdpr/). In GDPR, such data is called "personal data".
```

File: pages/your_problem/sensitive_data.md [hunk 2]
```diff
@@ -10,18 +10,22 @@ page_tag: sensitive
...
+* Confusingly, in the context of GDPR "sensitive data" is a subclass of "personal data" that is potentially even more harmful (defined in Article 9), and GDPR prescribes very strict rules for dealing with this data. Note that this page is concerned with the broader definition of "sensitive data". GDPR calls personal data sensitive consisting of racial or ethnic origin, political opinions, religious or philosophical beliefs, or trade union membership, genetic data, biometric data, data concerning health or data concerning a natural person's sex life or sexual orientation.
```
Comment (nsoranzo) [inline pages/your_problem/sensitive_data.md:22]:
```suggestion
* Confusingly, in the context of GDPR "sensitive data" is a subclass of "personal data" that is potentially even more harmful, and GDPR prescribes very strict rules for dealing with this data. Article 9 of GDPR defines sensitive data as personal data consisting of racial or ethnic origin, political opinions, religious or philosophical beliefs, or trade union membership, genetic data, biometric data, data concerning health or data concerning a natural person's sex life or sexual orientation. Note that this page is concerned with the broader definition of "sensitive data".
```

**Review by bedroesb — APPROVED** [review]

**Comment by bedroesb** [comment]
@nsoranzo thanks for your contributions! Do you want to be listed as contributors of this page?

**Comment by bedroesb** [comment]
If so I will add you!

**Comment by nsoranzo** [comment]
@bedroesb Not necessary, thanks for asking!

**Pull request merged**